test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Files that should be ignored by git must be placed in the .gitignore file, in order not to clutter the remote repo with information that we don't need there.
- [ ] For this reason, as it was a project requirement, kindly I ask, place this file in the .gitignore.